### PR TITLE
Whitelists dbl-tempo

### PR DESCRIPTION
**Pull request by pritishpai**
## Changes
Whitelists dbl-tempo

### Linked issues
Partly resolve #1931

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -2308,6 +2308,25 @@
     "dbignite.readers": [],
     "dbignite.version": []
   },
+  "dbl-tempo": {
+    "tempo": [],
+    "tempo.interpol": [],
+    "tempo.intervals": [],
+    "tempo.io": [
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2315]: this one is a false positive

File: src/databricks/labs/ucx/source_code/known.json [hunk 2]
```diff
@@ -2308,6 +2308,25 @@
...
+      {
+        "code": "cannot-autofix-table-reference",
+        "message": "Can't migrate table_name argument in ''optimize {} zorder by {}'.format(tabName, '(' + ','.join(partitionCols + optimizationCols) + ')')' because its value cannot be computed"
+      }
+    ],
+    "tempo.resample": [],
+    "tempo.tsdf": [
+      {
+        "code": "direct-filesystem-access",
+        "message": "The use of direct filesystem references is deprecated: /"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2325]: please create a better description of this error

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/source_code/known.json (hunk 2 above)
Comment (pritishpai) [reply src/databricks/labs/ucx/source_code/known.json]: removed the deprecated lib for now

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**